common/service.go

```diff
@@ -0,0 +1,65 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+)
+
+type (
+	Service  int32
+	Services []Service
+)
+
+const (
+	EmptyService Service = iota
+	StarWarsService
+	BaseService
+	BTCService
+	ETHService
+	GAIAChainRPCArchiveService
+)
+
+var ServiceLookup = map[string]int32{
+	"unknown":   0,
+	"swapi.dev": 1, // star wars API for development purposes
+	"github.com/arkeonetwork/arkeo-mainnet-fullnode": 2,
+	"btc-mainnet-fullnode":                           3,
+	"eth-mainnet-fullnode":                           4,
+	"gaia-mainnet-rpc-archive":                       5,
+}
+
+func (c Service) String() string {
```

Comment:
any reason to use the switch here vs a 2nd map like ServiceLookup above?
```
var RLookup = map[Service]string{
	StarWarsService:            "swapi.dev",
        ...
}

func (c Service) String() string {
	if r, ok := RLookup[c]; ok {
		return r
	}
	return "unknown"
}
```

Reply:
fixed in. [04fc4ce](https://github.com/arkeonetwork/arkeo/pull/134/commits/04fc4ce1776134be3686a3f37afd46e74f91bf2d)